ajax/libs/videojs-overlay/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "videojs-overlay",
+  "description": "A plugin to display simple overlays during video playback.",
+  "author": "Brightcove, Inc.",
+  "license": "Apache-2.0",
+  "keywords": [
+    "videojs",
+    "videojs-plugin"
```

Comment:
@sufuf3 @dakshshah96 I think we can add more keywords here next time like `overlay`